exercises/all-your-base/all_your_base_test.rb

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env ruby
 require 'minitest/autorun'
 require_relative 'all_your_base'
 
-# Common test data version: 1.0.0 39de23b
+# Common test data version: 1.1.0 c4d8d95
```

Comment:
Sorry, I don't think I understand.
Is there a reason to run bin/generate, but revert all the tests that were changed?